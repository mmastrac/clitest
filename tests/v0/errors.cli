#!/usr/bin/env clitest --version=0

$ exit 1
%EXIT 1

$ exit 1
%EXIT any

$ exit 0
%EXIT any

$ echo "a"
%EXPECT_FAILURE
! b
