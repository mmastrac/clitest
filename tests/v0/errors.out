Running <root>/errors.cli ...

exit 1
---
(no output)
---
✅ OK (exit status: 1)

exit 1
---
(no output)
---
✅ OK (exit status: 1)

exit 0
---
(no output)
---
✅ OK

printf "a\n"
---
a
---
✅ OK

<root>/errors.cli PASSED
